extension/src/webviewMessageFunctions.ts

```diff
@@ -67,5 +66,11 @@ export const executeRemainingHandlers = (message: any, webview: vscode.Webview)
         });
       });
       break;
+
+    case 'registerEmail':
```

Comment:
aren't we supposed to add it to the RESPONSE_PROVIDER?

Reply:
We can. Will need to refactor handleGetExtensionState() from Sidebar.svelte to a helper class so any svelte component can call it.

Reply:
I modified handleExtensionState in the other PR. I can address this comment later on.